config/bidderinfo_test.go

```diff
@@ -404,7 +407,7 @@ func TestBidderInfoValidationNegative(t *testing.T) {
 				},
 			},
 			[]error{
-				errors.New("at least one of capabilities.site or capabilities.app must exist for adapter: bidderA"),
+				errors.New("at least one of capabilities.site or capabilities.app or capabilities.dooh must exist for adapter: bidderA"),
 			},
 		},
```

Comment:
Please add a test case to `TestBidderInfoValidationPositive` to cover the call to `validatePlatformInfo`. This can be a copy of "One bidder incorrect capabilities for app" / "One bidder incorrect capabilities for site".